Tests/SwiftProtobufTests/Test_Struct.swift

```diff
@@ -143,6 +143,41 @@ class Test_JSON_ListValue: XCTestCase, PBTestHelpers {
         assertJSONDecodeSucceeds("[true]") {$0.values == [Google_Protobuf_Value(boolValue: true)]}
     }
 
+    func test_JSON_nested_list() throws {
+        let depths = [
```

Comment:
Since you avoid hardcoding the limit when do the parse test, maybe something like:

```
let depthLimit = JSONDecodingOptions().messageDepthLimit
let depths = [
            // right around the limit
            depthLimit - 1, depthLimit, depthLimit + 1,
            // Really big, should fail cleanly (not crash)
            1000,10000,100000,1000000,
]
```

Reply:
Done.